### ISSUE TITLE
Cannot currently launch the game directly from Steam

### ISSUE THREAD

**Issue by snoopy213** [FloatingOriginStudios/BalsaIssues#23]
For me to launch the game, I have to directly launch the balsa.exe from the game files. Attempting to launch from Steam yields a crash from Unity (screenshot attached).

Have validated files, and reinstalled, but the issue persists. Unsure how to reproduce/point to a cause, I believe it might be me playing the game in 1920x1080 in fullscreen, as these are the only settings I changed after booting successfully from Steam the first time.

Specs: Windows 10 x64, Intel i7, GTX 1070

Screenshot: 
![image](https://user-images.githubusercontent.com/89785118/131376659-f3d5e929-681d-4bb6-8921-ff0d3c37a9a0.png)

**Comment by PsykomantaBrain** [comment]
I have to say that is a new one to me. 

Can you post the log file so we can see what's going on? It should be in "C:\Users\ [you] \AppData\LocalLow\Floating Origin Interactive\Balsa\Player.log"

**Comment by snoopy213** [comment]
[Player.log](https://github.com/FloatingOriginInteractive/BalsaIssues/files/7078588/Player.log)

Attached above. I can still play, just have to launch manually. V weird

**Comment by PsykomantaBrain** [comment]
Huh, it crashed right away, even before it started to initialise the game proper.

Do you have steam running under some compatibility mode, or maybe as administrator? Possibly some antivirus/antimalware software running?

The errors I'm seeing on the log are very generic and don't mean much by themselves, but given it only happens when launching through steam, I suspect it might be inheriting some application setting from the parent process (steam) that launched it.

**Comment by snoopy213** [comment]
Nope, none of the above. No off-the-shelf antivirus/antimalware besides MS Defender that came preinstalled with Win10. 

For now, I'll keep using the workaround. Might be an individual issue, just wanted to put this on your radar in case anyone sees similar issues. Thanks for your help 😄
